config/site_settings.yml

```diff
@@ -411,6 +411,22 @@ basic:
   show_user_menu_avatars:
     client: true
     default: false
+  extended_site_description:
+    default: ""
+    max: 10_000
+    hidden: true
```

Comment:
I think since these are hidden now you need to add `include_hidden: true` at `Admin::SiteSettingsController#index`

Reply:
Otherwise the page is currently broken

Reply:
NVM this I saw + responded to your other comment about include_hidden :)